### ISSUE TITLE
add AVS into Android mobile phone?

### ISSUE THREAD

**Issue by wantitmore** [willblaschko/AlexaAndroid#125]
Is your project an application? I want to know how to add AVS into my android devices rather than build an application in my phone. do you know how to work it? thx!

**Comment by syedtalha** [comment]
Yes, there is  an app already  available[here](https://play.google.com/store/apps/details?id=agency.rain.android.alexa&hl=en)

**Comment by wantitmore** [comment]
@syedtalha  something wrong go with me. Is your project  similar with the link you offer? Now I want to build an application which be integrated with AVS? Does your project can meet my requirment? thx

**Comment by syedtalha** [comment]
I'm not sure I'm following what you're trying to say. Basically this library can be integrated into any project, including yours. So if your requirement is to have Alexa inside your app, then yes, this library meets your requirements

**Comment by wantitmore** [comment]
@syedtalha  I have downloaded your sample app in google play. My question is: can I use your sample app or library to control other smart devices(such as light,switch) just like amazon echo do.
In other words, I don't have echo dot, what I have is an android phone and your project,can I control smart devices? thx

**Comment by syedtalha** [comment]
@wantitmore  Yes definitely. You can use it just like an echo device, except the calling feature. 

And BTW, I do not have any part in developing that app. I am just a user.

**Comment by wantitmore** [comment]
thanks  for your apply.do u know how to connect the devices(such a light to be controlled turn on/off)?or is it possible to do it?

**Comment by syedtalha** [comment]
You can hook up your light to this alexa compatible socket. https://m.lightinthebox.com/smart-remote-control-wifi-plug-socket-support-alexa-voice-control-for-home-appliances_p6118171.html

**Comment by wantitmore** [comment]
the link you offer should have an echo dot.  I don't know how to connect the socket to my android project rather than any other alexa-able hardware devices. thx~

**Comment by willblaschko** [comment]
@wantitmore this library doesn't do direct connection to devices, it leverages Alexa's AVS library, so all commands need to be through the Alexa cloud. This library/app essentially functions like an Echo device on your phone (with some strong limitations).

If you want to control individual devices, not through an existing framework, at a high level, you should research network discovery and communication protocols.

**Comment by wantitmore** [comment]
@willblaschko so,do you think it is possible to control other devices?

**Comment by wantitmore** [comment]
and your sample app didn't use avs-device-sdk , so what 's the role of avs-device-sdk?thx
